src/ui/FacetValueSuggestions/FacetValueSuggestions.ts

```diff
@@ -268,10 +269,27 @@ export class FacetValueSuggestions extends Component {
   }
 
   private mapFacetValueSuggestion(resultToShow: IFacetValueSuggestionRow, omnibox: Omnibox) {
-    return <IOmniboxSuggestion>{
+    const field: ISuggestionField = {
+      name: this.options.field.toString(),
+      value: resultToShow.value
+    };
+
+    const suggestion = <IOmniboxSuggestion>{
```

Comment:
Why use a cast instead of typing the suggestion variable? e.g.

```
const suggestion: IOmniboxSuggestion = { 
    ...
};
```

Reply:
Woops, I accidentally left over this cast when moving some code. Fixing it.